.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+## Proposed changes
+
+<!--
+Describe an overview of the changes, why they are useful, and any comments on the implementation so that it is easy for reviewers to understand the PR.
+-->
+
+### Types of changes:
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
```

Comment:
What does a breaking change even mean for us?  I've generally seen it in the context of certain changes to a stable public interface, but we don't really have any sort of stable public interface.

And if the classification does make sense, shouldn't there be an option to select for a breaking change?

Reply:
offline discussion we had:

Currently don't have an interface, will need to define it once we have a "stable" version (need to define that too).